cumulusci.yml

```diff
@@ -406,7 +417,7 @@ flows:
             6:
                 task: deploy_qa_config
             7:
-                task: deploy_health_check_config 
+                task: deploy_health_check_config
```

Comment:
Will this need to be added to K12?

Reply:
No. This task will disappear before we go to a managed version. Right now it's a workaround to expose the top level health check component when working in an unmanaged context without having it exposed in our managed code.